python311Packages.langsmith: 0.0.14 -> 0.0.24

fabaff opened the pull request:
Diff: https://github.com/langchain-ai/langsmith-sdk/compare/refs/tags/v0.0.14...v0.0.24

Changelog: https://github.com/langchain-ai/langsmith-sdk/releases/tag/v0.0.24

## Description of changes

<!--
For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

fabaff committed:
python311Packages.langsmith: add changelog to meta

fabaff committed:
python311Packages.langsmith: disable on unsupported Python releases

fabaff committed:
python311Packages.langsmith: 0.0.14 -> 0.0.24

Diff: https://github.com/langchain-ai/langchainplus-sdk/compare/refs/tags/v0.0.14...v0.0.24

Changelog: https://github.com/langchain-ai/langsmith-sdk/releases/tag/v0.0.24

natsukium reviewed (commented):
Thanks for your PR.
Since the repo name has changed, can you also update the `repo` and `homepage`?

fabaff committed:
python310Packages.langchain: 0.0.263 -> 0.0.268

Diff: https://github.com/hwchase17/langchain/compare/refs/tags/v0.0.263...v0.0.268

Changelog: https://github.com/hwchase17/langchain/releases/tag/v0.0.268

fabaff committed:
python311Packages.langsmith: update repo details

matthiasbeyer commented:
Result of `nixpkgs-review pr 249977` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages failed to build:</summary>
  <ul>
    <li>pentestgpt</li>
    <li>pentestgpt.dist</li>
    <li>python310Packages.langchain</li>
    <li>python310Packages.langchain.dist</li>
  </ul>
</details>
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>python310Packages.langsmith</li>
    <li>python310Packages.langsmith.dist</li>
    <li>python311Packages.langsmith</li>
    <li>python311Packages.langsmith.dist</li>
  </ul>
</details>

natsukium reviewed (approved):
LGTM, thanks!
